pnet_datalink/src/winpcap.rs

```diff
@@ -324,17 +324,17 @@ pub fn interfaces() -> Vec<NetworkInterface> {
         }
     }
 
-    let mut buf = [0u8; 4096];
+    const IFACE_BUF_LEN: usize = 4096;
+    let mut buf = vec![0u8; IFACE_BUF_LEN];
```

Comment:
I wouldn't bother with the constant - it's only used here, so doesn't really add anything